graphql-kotlin-federation/src/main/kotlin/com/expedia/graphql/federation/FederatedSchemaGeneratorHooks.kt

```diff
@@ -59,9 +59,11 @@ open class FederatedSchemaGeneratorHooks(private val federatedTypeRegistry: Fede
             federatedQuery.field(entityField)
 
             // SDL returned by _service query should not contain directives or new scalars
-            val sdl = originalSchema.print()
+            val sdl = originalSchema
+                .print(includeDefaultSchemaDefinition = false)
                 .replace(directiveRegex, "")
                 .replace(scalarRegex, "")
+                .replace("type Query", "type Query @extends")
```

Comment:
I don't think this makes much sense, each service could define number of root queries that are not federated. Since this would apply to all federated schemas then there wouldn't be a base Query type anywhere, e.g.

service A - defines root query to resolve base type with @key 
service B - extends base type from service A

In the above situation both `Query` types would be marked as `extended` which is wrong